src/container/MovieContainer.js

```diff
@@ -15,12 +16,21 @@ const MovieContainer = ({ movies }) => {
     )
   })
 
-  return (
-    <div className='movie-container'>
-      {moviePosters}
-    </div>
-  )
-
+  if (isLoaded) {
+    return (
+      <div>
+        <Form filterMovies={filterMovies} />
+        <div className='movie-container'>
+          {moviePosters.length ? moviePosters : <h2>No results</h2>}
```

Comment:
We might need a className on this h2 for styling purposes, I can do this in my styling branch